src/app/helptext/jails/jails-add.ts

```diff
@@ -531,6 +531,9 @@ allow_tun_tooltip: T('Reveal <i>tun</i> devices for the jail with an \
 individual devfs ruleset. Allow the creation of <i>tun</i> devices in \
 the jail.'),
 
+rtsold_placeholder: T('rtsold'),
```

Comment:
```suggestion
rtsold_placeholder: T('rtsold With Autoconfigure IPv6'),
```

Reply:
Hi Tim, could you double check with Brandon about the placeholder changes? thanks